tests/e2e/default/contacts/muting-contact-using-form.wdio-spec.js

```diff
@@ -0,0 +1,75 @@
+const utils = require('../../../utils');
+const commonPage = require('../../../page-objects/default/common/common.wdio.page');
+const loginPage = require('../../../page-objects/default/login/login.wdio.page');
+const userFactory = require('../../../factories/cht/users/users');
+const placeFactory = require('../../../factories/cht/contacts/place');
+const personFactory = require('../../../factories/cht/contacts/person');
+const modalPage = require('../../../page-objects/default/common/modal.wdio.page');
+const contactPage = require('../../../page-objects/default/contacts/contacts.wdio.page');
+const genericForm = require('../../../page-objects/default/enketo/generic-form.wdio.page');
+const deathReportForm = require('../../../page-objects/default/enketo/death-report.page');
+const undoDeathReportForm = require('../../../page-objects/default/enketo/undo-death-report.page');
+
+describe('Mute/Unmute contacts using a specific form.', () => {
+  const places = placeFactory.generateHierarchy();
+  const healthCenter = places.get('health_center');
+  const offlineUser = userFactory.build({ place: healthCenter._id, roles: ['chw'] });
+  const person = personFactory.build({ parent: {_id: healthCenter._id, parent: healthCenter.parent} });
+  const mutePerson = personFactory.build({ 
+    patient_id: 12345, 
```

Comment:
Is this patient_id value really need to be set? or you can use the default value set in `personFactory`

Reply:
I set a different value for the `patient_id` because the `person` was already using the default value that was assigned in the `personFactory`

Reply:
Oh! ok I get it. Thanks.